src/quo2/components/profile/profile_card/view.cljs

```diff
@@ -4,77 +4,102 @@
             [quo2.components.icon :as icon]
```

Comment:
Should we sort namespaces alphabetically?

Reply:
Hi, I don't know, we don't have any universal pattern for sorting `:requires`. I usually sort them in this stair-like pattern while creating a **new** [namespace](https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/shell/bottom_tabs.cljs) (maybe just because it looks clean), but they always get [reordered](https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/shell/home_stack.cljs) after some time. So probably don't matter much.